Tests/Adapter/VarnishCacheTest.php

```diff
@@ -81,4 +81,33 @@ public function testValidToken()
         $cache = new VarnishCache('token', array(), $router, 'ban', $resolver);
         $cache->cacheAction($request);
     }
+
+    public function testRunCommand()
+    {
+        $tmpFile = tempnam(sys_get_temp_dir(), '');
```

Comment:
Please specify "sonata-cache" as second argument.
Rationale behind this : on a production server of my company, `/tmp` often gets full because of temporary files that clutter it. It's harder to know which application a "temporary" files come from if you don't have this prefix.